include/zephyr/net/tftp.h

```diff
@@ -53,41 +44,133 @@ struct tftpc {
 #define TFTPC_REMOTE_ERROR       -4
 #define TFTPC_RETRIES_EXHAUSTED  -5
 
-/* @brief This function gets "file" from the remote server.
+/**
+ * @brief TFTP Asynchronous Events notified to the application from the module
```

Comment:
I'm not sure if "asynchronous" is the right word here, it could suggest the API is asynchronous, while it's not. I think simple "TFTP events" would do.

Reply:
Yes, misleading wording. I'll update